sgx-tools/gen-qe-target-info.go

```diff
@@ -27,12 +37,35 @@ For example, save the target information file about Quoting Enclave retrieved fr
 			logrus.SetLevel(logrus.DebugLevel)
 		}
 
-		ti, err := intelsgx.GetQeTargetInfo()
+		var ti []byte
+		var err error
+
+		serviceType := context.String("serviceType")
+		AkId, err := GetAttKeyId(serviceType, context.String("AkId"))
```

Comment:
Essentially, quote type implies the registered and known attestation key ID, and AkId specified from cmdline is the private attestation key ID used by special scenario, or intending to use raw attestation key ID hex string for testing purpose. So they belong to the same function with different style of parameter. I suggest using two APIs to distinguish them:

- For 99% cases: GetQeTargetInfo(quoteType)
- For rare cases: GetQeTargetInfoByAttestationKeyId(akId)